docs/code-samples/instruments/get-dependencies.bat

```diff
@@ -0,0 +1,21 @@
+:: Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+:: SPDX-License-Identifier: Apache-2.0
+
+@echo off
+
+:: Create .lib directory
+if exist ".\.lib\" rmdir /s /q .lib
+mkdir .\.lib
+
+:: Get the dependency list
+echo Downloading the list of dependencies
+for /f "tokens=2" %%a IN ('findstr "^version" daml.yaml') DO (set version=%%a)
+
+curl -Lf# "https://raw.githubusercontent.com/digital-asset/daml-finance/main/docs/code-samples/getting-started-config/%version%.conf" -o .lib/%version%.conf
```

Comment:
if you add a new config file, then here you should point to the new one. I would be fine with using the same though